universal-application-tool-0.0.1/app/services/question/QuestionDefinition.java

```diff
@@ -117,6 +122,42 @@ public String getName() {
     return this.name;
   }
 
+  /**
+   * A repeater question is used to enumerate a variable list of user-defined identifiers for a
+   * repeated entity (e.g. children, or household members).
+   *
+   * @return true if this is a repeater question.
+   */
+  public boolean isRepeater() {
+    return getQuestionType().equals(QuestionType.REPEATER);
+  }
+
+  /**
+   * See {@link #getRepeaterId()}.
+   *
+   * @return true if this is a repeated question.
```

Comment:
Nit (blocking): Would you add that a **repeated** question is a child of a *repeater* question? The names are too similar and make me wonder if we should give a different name to either.

Reply:
The `See {@link #getRepeaterId()}` has all the context there. Should I duplicate it?

Reply:
Ah, that's good.